docs/instructions.adoc

```diff
@@ -74,8 +74,8 @@ If you wish to use {uri-oci-kms}[OCI KMS] to encrypt Kubernetes secrets, the fol
 * link:#adding-the-bastion-host[bastion must be enabled]
 * link:#enabling-instance_principal-on-the-operator-host[operator instance_principal must be enabled]
 
-* use_encryption must be set to _true_
-* kms_key_id must be provided
+* use_cluster_encryption must be set to _true_
```

Comment:
Please use backticks when documenting variables: `my_variable`

Reply:
ok.

Reply:
backtick wont work in this place. Backtick was not there as well before its just a variable name change.